backend/battle/battles/base_stats.py

```diff
@@ -44,10 +44,10 @@ def get_pokemons_team(battle, trainer):
     }
     team = Team.objects.prefetch_related('pokemons').filter(battle=battle, trainer=trainer)
     if team:
-        pokemons_team_query = team[0].pokemons.all()
-        pokemons_team["pokemon_1"] = pokemons_team_query[0]
-        pokemons_team["pokemon_2"] = pokemons_team_query[1]
-        pokemons_team["pokemon_3"] = pokemons_team_query[2]
+        pokemon_team = team[0].teams.all().order_by('order').prefetch_related('pokemon')
```

Comment:
This is a bit strange because you're not using the `Team` instance anymore.
I think you can query instead directly from `PokemonTeam` in line 45. Please check if you can make this refactor.

Reply:
I need to use the Team because in line 45 I verify if a team exists, if not, it'll return false.
If a team exists, I get the data of pokemon_team filtering by the team

Reply:
But if the PokemonTeam for that user exists, then the Team exists.

So consider if you can filter by PokemonTeam crossing Team, like this: `PokemonTeam.objects.filter(team__battle=battle, team__trainer=trainer).select_related('pokemon')`

Reply:
It works, thanks!